js/actions/windowActions.js

```diff
@@ -213,6 +213,9 @@ const WindowActions = {
     if (frameOpts.location && UrlUtil.isURL(frameOpts.location)) {
       frameOpts.location = UrlUtil.getUrlFromInput(frameOpts.location)
     }
+    // default the title to the location
+    frameOpts.title = frameOpts.title || frameOpts.location
```

Comment:
I don't think this is correct, why is it needed? 
We render title based on if there is no title then use location.